vendor/calcite-1_22_0/build.gradle

```diff
@@ -0,0 +1,111 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * License); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins { id 'org.apache.beam.vendor-java' }
+
+description = "Apache Beam :: Vendored Dependencies :: Calcite 1.22.0"
+
+group = "org.apache.beam"
+version = "0.1"
+
+def calcite_version = "1.22.0"
+def avatica_version = "1.16.0"
+def prefix = "org.apache.beam.vendor.calcite.v1_22_0"
+
+List<String> packagesToRelocate = [
+        "antlr",
+        "com.beust",
+        "com.esri",
+        "com.fasterxml",
+        "com.google.common",
+        "com.google.gson",
+        "com.google.inject",
+        "com.google.protobuf",
+        "com.google.thirdparty",
+        "com.jayway.jsonpath",
+        "com.yahoo",
+        "javax",
+        "net.minidev",
+        "net.sf",
+        "org.antlr",
+        "org.aopalliance",
+        "org.apache.calcite",
+        "org.apache.commons",
+        "org.apache.http",
+        "org.apache.tool",
+        "org.apiguardian.api",
+        "org.codehaus",
+        "org.easymock",
+        "org.eclipse.jetty",
+        "org.json",
+        "org.objectweb",
+        "org.objenesis",
+        "org.pentaho",
+        "org.slf4j",
```

Comment:
If you relocate logging, you will lose all logging from this vendored library.

Consider not relocating them and marking them as runtime depedencies.

Reply:
move logging library as runtime dependencies